yt_dlp/extractor/instagram.py

```diff
@@ -234,7 +234,9 @@ def _real_extract(self, url):
                 media, lambda x: x['edge_media_to_caption']['edges'][0]['node']['text'],
                 compat_str) or media.get('caption')
             title = media.get('title')
-            thumbnail = media.get('display_src') or media.get('display_url')
+            display_resources = media.get('display_resources')
+            if not display_resources:
+                display_resources = [{'url': media.get('display_src') or media.get('display_url')}]
```

Comment:
Did u mean to do this?
```suggestion
                display_resources = [{'src': media.get('display_src') or media.get('display_url')}]
```